### PR TITLE
Remove enum from catalogPricingTier

### PR DESCRIPTION
**Pull request by zhangyd2015**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jianghaolu** [comment]
What's the reason for this change? How can customers discover what string to provide?

**Comment by zhangyd2015** [comment]
> What's the reason for this change? How can customers discover what string to provide?

This is actually a increasing list which comes from Azure SQL database. If we make it enum here, we need to  update the list from time to time, so we decided to just change it to string and document it with a link from Azure SQL Database site. The current enum list is actually incorrect and out of date.

**Review by jianghaolu — APPROVED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json [hunk 1]
```diff
@@ -187,17 +187,7 @@
...
-            "Basic",
-            "Standard",
-            "Premium",
-            "PremiumRS"
-          ],
-          "x-ms-enum": {
-            "name": "IntegrationRuntimeSsisCatalogPricingTier",
-            "modelAsString": true
-          }
+          "type": "string"
```
Comment (jianghaolu) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: This is not allowed though. This is a stable API version so no breaking changes can be introduced.

**Review by jianghaolu — CHANGES_REQUESTED** [review]
Sorry for the delay and thanks for the explanation.

You can keep the changes for the preview spec but you cannot change the stable spec. You will have to include this breaking change in the next API version update.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json (hunk 1 above)
Comment (zhangyd2015) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: @jianghaolu I revoked the changes in stable API version.

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5428 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5428/commits/98414334139d11c9997ea9ebcc32894083a6f3e8

**Review by hvermis — APPROVED** [review]

**Review by hvermis — APPROVED** [review]

**Comment by zhangyd2015** [comment]
@jianghaolu Could you please merge this PR if there is no further comments? Thanks.

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**